src/client/components/app/ApyTooltipData/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useAppTranslation } from '@src/client/hooks';
+import { getConstants } from '@src/config/constants';
+import { formatApy } from '@src/utils';
+import { Apy } from '@yfi/sdk';
+import { FC } from 'react';
+import styled from 'styled-components';
+
+const StyledTooltipTable = styled.table`
+  & > tr > td {
+    font-size: 1.2rem;
+    &:first-of-type {
+      padding-right: 10px;
+    }
+  }
+`;
+
+export interface ApyTooltipDataProps {
+  apy: Apy;
+  address: string;
+}
+
+const ApyTooltipData: FC<ApyTooltipDataProps> = ({ apy, address }) => {
+  const { t } = useAppTranslation(['common', 'vaults']);
+  const { CONTRACT_ADDRESSES } = getConstants();
+  const { YVECRV, YVBOOST, PSLPYVBOOSTETH } = CONTRACT_ADDRESSES;
+  const isBackScratcher = address === YVBOOST || address === YVECRV || address === PSLPYVBOOSTETH;
```

Comment:
this should be isLab. The three of those assets are Labs. YVECRV its called backscratcher

Reply:
Fixed! Seems like for YVBOOST and PSLPYVBOOST composite is coming as null for now so backscratcher is the only that is getting a different casing